src/components/Card/Card.jsx

```diff
@@ -38,6 +52,13 @@ function Card({
           {articleTitle}
         </p>
       </a>
+      <button
+        aria-label="openAI-summary"
+        className="absolute bottom-5 right-5 text-gray hover:text-black"
+        onClick={handleClickShowSummary}
```

Comment:
```suggestion
        onClick={handleShowSummaryClick}
```

다른 코드와 일관성을 유지하기 위해 이렇게 변경해 주시면 좋을 것 같습니다!
아래 글도 한번 읽어보시면 좋을 것 같아요.
[React의 이벤트 핸들러 네이밍](https://jaketrent.com/post/naming-event-handlers-react/)